### ISSUE TITLE
All modals should be closeable by clicking gray backdrop

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos#5367]
## Detailed Description
Many modals, e.g. NML import can be closed when clicking outside of the modal area, e.g. on the gray back drop. This does not work for all modals, though.
I noticed this on the user list view for 
 - Change Experiences
 - Change Teams & Permissions
and on dashboard for the “archive all” confirm modal. There are probably others.

## Context
The gray area around a modal offers a huge click target which allows for faster navigation. Also, this being possible for *some* modals but not all is confusing

**Comment by philippotto** [comment]
Just some food for thought: I think we should check twice before doing this for all modals. For example, doing some batch editing of experience domains can take several minutes within the modal. If one clicks accidentally on the "huge click target", this can be very annoying to the user as all the work is lost then. I even vaguely remember a "bug report" for this (not sure, though).

**Comment by fm3** [comment]
Fair point :thinking: Do you know if there is an easy way to search the code for the information which modals have this and which don’t? Then we could discuss if it should be added to any specific one. However, that’s a low priority then.

**Comment by hotzenklotz** [comment]
The antd modal property is called `maskClosable`. A quick search reveals these components:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/1105056/197197378-0d133d83-2d04-4ade-b60c-ec49b292066d.png">

I vote to make the following closable:
- downsample_volume_modal
- experiences_modal_view
- dataset_add_view
